run_kraken.py

```diff
@@ -348,6 +349,9 @@ def main(cfg):
                         elif scenario_type == "network_chaos":
                             logging.info("Running Network Chaos")
                             failed_post_scenarios, scenario_telemetries = network_chaos.run(scenarios_list, config, wait_duration, kubecli, telemetry_k8s)
+                        elif scenario_type == "service_hijacking":
+                            logging.info("Running Service Hijacking Chaos")
+                            service_hijacking_plugin.run(scenarios_list,kubecli)
```

Comment:
Hitting TypeError: run() missing 1 required positional argument: 'telemetry'. 

Has to be " service_hijacking_plugin.run(scenarios_list,kubecli, telemetry_k8s)". Thoughts?

Reply:
Sorry I was pushing while you were testing.

Reply:
Np at all @tsebastiani! It's working as expected!